src/styles/_mixins.scss

```diff
@@ -0,0 +1,19 @@
+@mixin onTablet {
+  @media (min-width:744px) {
+    @content;
+  }
+}
+
+@mixin onDesktop {
+  @media (min-width:1024px) {
+    @content;
+  }
+}
+
+@mixin title {
+  font-weight: 700;
```

Comment:
You may want to use a variable for the title font size (e.g., $title-font-size) to maintain consistency across your project and make it easier to update. 📏